doc/source/tutorial/stats/sampling.rst

```diff
@@ -85,6 +85,7 @@ Methods for continuous distributions   Required Inputs  Optional Inputs  Setup S
 :class:`~TransformedDensityRejection`  pdf, dpdf        none             slow         fast
 :class:`~NumericalInverseHermite`      cdf              pdf, dpdf        (very) slow  (very) fast
 :class:`~NumericalInversePolynomial`   pdf              cdf              (very) slow  (very) fast
+:class:`~SimpleRatioUniforms`          pdf              none             fast         slow
```

Comment:
At some point I think it would be nice to add relative performance information. It would give a better sense of what is behind these adjectives.